dashboard-server/queries/responses/response.queries.js

```diff
@@ -13,7 +13,10 @@ async function all() {
        SELECT DISTINCT ON (1) userid, timestamp AS last_timestamp, response AS last_response, surveyid
        FROM   responses
        ORDER  BY 1,2 DESC
-       ) l USING (userid)`;
+       ) l
+    USING (userid)
+    ORDER BY first_timestamp DESC
```

Comment:
This is basically a copy and paste from, https://github.com/vlab-research/fly/pull/59#discussion_r772623607